### ISSUE TITLE
reset initial pose through ros service

### ISSUE THREAD

**Issue by RamanaBotta** [PRBonn/kiss-icp#237]
Hello, 
### About
Since kiss-icp is prone to accumulation of positional errors over time, having something like a service call to reset odometry would really help when working with other sensors, especially sensors like GPS (globally consistent).
### Existing solutions
This kind of feature is available with Zed Cemera positional tracking, Where through a service call it we can reset the start position of odometry [Ref](http://wiki.ros.org/zed-ros-wrapper).

**Comment by nachovizzo** [comment]
I agree this would be great to have. Would you like to contribute it to the project? I can guide you a bit once you have a prototype ready ;)

**Comment by RamanaBotta** [comment]
I am happy to contribute :)

**Comment by RamanaBotta** [comment]
I found a related code snippet in the Zed wrapper.
https://github.com/stereolabs/zed-ros-wrapper/blob/71eb2bb434f059e17191503b707267938f5a1b7f/zed_nodelets/src/zed_nodelet/include/zed_wrapper_nodelet.hpp#L408

**Comment by RamanaBotta** [comment]
Hello, please find the pull request regarding this issue.
